### PR TITLE
add a bit about developing in the open

### PR DESCRIPTION
**Pull request by dotcode**
Addresses @KaimingTao's question in https://github.com/springernature/frontend/pull/65#issuecomment-240372653

### CODE REVIEW

**Comment by andrewmee** [comment]
👍

**Comment by benjclark** [comment]
👍

**Comment by KaimingTao** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+
 
 #Contributing
 
 To contribute please create a new branch for each section, then create a pull request to merge your changes in.
 
 Please keep discussion inside the issues and pull requests, avoiding Slack, hallway conversations etc. Remember that this repo is public, and the discussions we have can be of benefit to people apart from us.
+
+[this contribution]: https://github.com/springernature/frontend/pull/48#issuecomment-236139605
+[Changing the laws of engineering with pull requests]: https://github.com/springernature/frontend/pull/48#issuecomment-236139605
```
Comment (KaimingTao) [inline README.md]: The link of "Changing the laws of engineering with pull requests" makes me confused. It's the same as "this contribution". I think it should be linked to somewhere else?...

**Pull request merged**